config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -55,6 +55,9 @@ en:
             multiplier: 'Multiplier'
             question: 'Question'
             grade_summary: 'Grade Summary'
+            publish_grade_hint: >
+              The grades are not shown to the student. You can publish the grades at the submissions
```

Comment:
Saves the current grading. Note that students cannot see these grades until they are published. This option can be found at the submissions page of current assessment.